src/components/resultsSidebar.tsx

```diff
@@ -22,69 +41,206 @@ const useStyles = makeStyles((theme) => ({
       // position: 'relative',
     },
   },
+  contentCard: {
+    paddingBottom: '0 !important',
+  },
   card: {
     margin: '1vw',
-    width: '33vw',
+    width: '31vw',
+    paddingBottom: 16,
     [theme.breakpoints.down('md')]: {
-      width: '96vw',
+      width: '94vw',
       margin: '2vw',
     },
   },
   sidebarMessage: {
     textAlign: 'center',
     padding: '3vw',
   },
+  listHeading: {
+    fontWeight: 'bold',
+    marginRight: 32,
+    whiteSpace: 'inherit',
+    lineHeight: 1.4,
+  },
+  placeContainer: {
+    position: 'relative',
+  },
+  placeImage: {
+    width: 100,
+    height: 100,
+    objectFit: 'cover',
+    float: 'left',
+    // paddingBottom: 16,
+    marginRight: 12,
+  },
+  innerRow: {
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    width: 'auto',
+    overflow: 'hidden',
+  },
+  favorite: {
+    position: 'absolute',
+    top: 7,
+    right: 7,
+    cursor: 'pointer',
+  },
 }));
 
+const SidebarTitle = ({ classes, text }: { classes: object; text: string }) => (
+  <Typography className={classes.sidebarMessage} variant="h5" component="h3">
+    {text}
+  </Typography>
+);
+
+export const PlaceCard = ({
+  place,
+  width,
+}: {
+  place: object;
+  width: number;
+}) => {
+  const classes = useStyles();
+  const { toggleFavorite } = useContext(StoreContext);
+  let ratings = Array.from({ length: 5 }, (v, i) => i);
+  const roundedRating = Math.floor(parseFloat(place?.rating));
+  ratings = ratings.map((val) => (roundedRating > val ? '+' : '-'));
+  const priceLevel = Array.from(
+    { length: parseInt(place?.price_level, 10) },
+    (v, i) => i,
+  );
+
+  return (
+    <div className={classes.placeContainer} style={{ width: width ?? 'auto' }}>
+      <img
+        alt={place?.name}
+        className={classes.favorite}
+        src={place?.favorite ? FavActive : FavInactive}
+        onClick={(e: MouseEvent) => {
+          e.stopPropagation();
+          toggleFavorite(place);
+        }}
+      />
+      <img
+        className={classes.placeImage}
+        alt={place.name}
+        src={
+          place?.photos?.[0]?.getUrl
+            ? place?.photos?.[0]?.getUrl({
+              maxWidth: 150,
+              maxHeight: 150,
+            })
+            : RestaurantFallback
+        }
+      />
+      <div>
+        <Typography className={classes.listHeading} variant="h6">
+          {place.name}
+        </Typography>
+        {place?.rating && place?.user_ratings_total && (
+          <div>
+            {ratings.map((isStar) => (
+              <img alt="star" src={isStar === '+' ? StarFilled : StarEmpty} />
+            ))}
+            {' '}
+            (
+            {place?.user_ratings_total?.toLocaleString('en-US')}
+            )
```

Comment:
My IDE and eslint rules are fighting each other for some reason on indentation :/